Examples/Swift/HeatmapExample.swift

```diff
@@ -15,53 +14,53 @@ class HeatmapExample: UIViewController, MGLMapViewDelegate {
         mapView.tintColor = .lightGray
         view.addSubview(mapView)
     }
-    
+
     func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
         // Parse GeoJSON data. This example uses all M1.0+ earthquakes from 12/22/15 to 1/21/16 as logged by USGS' Earthquake hazards program.
         guard let url = URL(string: "https://www.mapbox.com/mapbox-gl-js/assets/earthquakes.geojson") else { return }
         let source = MGLShapeSource(identifier: "earthquakes", url: url, options: nil)
         style.addSource(source)
-        
+
         // Create a heatmap layer.
         let heatmapLayer = MGLHeatmapStyleLayer(identifier: "earthquakes", source: source)
-        
+
         // Adjust the color of the heatmap based on the point density.
-        let colorDictionary : [NSNumber : UIColor] = [
-                                0.0 :  .clear,
-                               0.01 : .white,
-                               0.15 : UIColor(red:0.19, green:0.30, blue:0.80, alpha:1.0),
-                               0.5 : UIColor(red:0.73, green:0.23, blue:0.25, alpha:1.0),
-                               1 : .yellow
+        let colorDictionary: [NSNumber: UIColor] = [
```

Comment:
The indentation in `colorDictionary` and `magnitudeDictionary` isn’t consistent, re: first entry on new line.